secure-device-storage/sds-android/app/build.gradle

```diff
@@ -21,4 +21,5 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:25.1.1'
+    compile project(path: ':securedevicestorage')
```

Comment:
Fixed in commit: https://github.com/adorsys/secure-key-storage/pull/54/commits/825b5723dd621d327ac110cc7399f8372493cca4